.changeset/spotty-buses-drive.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-catalog': patch
+---
+
+Enable loading state for paginated catalog tables by passing isLoading prop
```

Comment:
Maybe add "`" around `isLoading` so that it passes the Vale check

Reply:
@drodil thanks! let me check it out 😄